db/fixtures/miq_product_features.yml

```diff
@@ -5123,6 +5123,54 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+
+# Embedded Automation Manager
+- :name: Embedded Automation Manager
+  :description: Everything under Ansible
+  :feature_type: node
+  :identifier: embedded_automation_manager
+  :children:
+  - :name: Repositories
+    :description: Everything under Repositorie
+    :feature_type: node
+    :identifier: configuration_script_source
```

Comment:
@lgalis I thought we had discussed that the "external" ones would be the ones without the prefix, and that "embedded" (because it's the abnormal one) would have the prefix of "embedded"?  This seems to be the opposite.